debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/ApicurioInfrastructure.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.debezium.testing.testcontainers;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import org.testcontainers.images.builder.ImageFromDockerfile;
+import org.testcontainers.lifecycle.Startables;
+
+import java.util.stream.Stream;
+
+/**
+ * Apicurio Testcontainers infrastructure handling.
+ */
+
+public class ApicurioInfrastructure extends GenericContainer<ApicurioInfrastructure> {
+
+    protected static final String DEBEZIUM_VERSION = "1.3.0.Final";
+    protected static final String APICURIO_VERSION = "1.3.2.Final";
+    private final GenericContainer<?> apicurioContainer;
+    private final ImageFromDockerfile apicurioDockerImage;
```

Comment:
```suggestion
    private final ImageFromDockerfile apicurioContainerImage;
```

We should generally use "container image" instead of "Docker image" (here, and elsewhere). Docker is just one particular implementation used for creating OCI-compliant container images.